python/pylibraft/pylibraft/common/handle.pyx

```diff
@@ -32,7 +32,7 @@ from .cuda import CudaRuntimeError
 
 
 cdef class DeviceResources:
-    """
+    '''
```

Comment:
This is super nit-picky - but the previous `"""` is inline with the `black` python styling we are using here (this would fail code formatting if it were a pure python file instead of a a cython file - and is only passing because black doesn't handle cython).

Reply:
yep, I'll change this. One of the many things I had tried to get the doctest strings works